code/modules/organs/cybernetics/upgrades/cybernetics.dm

```diff
@@ -0,0 +1,38 @@
+/obj/item/cybernetics
+	//This thing doesn't actually exist, just something for inheritance
+	var/required_type //If the upgrade is only for a specific organ ("internal_organ" for internal, "external_organ" for external), if not then keep it empty
+
+/obj/item/cybernetics/afterattack(var/obj/item/O, mob/user, proximity_flag)
+	if (!proximity_flag)
+		return
+
+	if(istype(O, /obj/item/organ))
+		var/obj/item/organ/I = O
+		var/datum/organ/organ = I.organ_type
```

Comment:
This is a typepath, not an organ datum.
If you want the organ datum, you need to access organ_data on the organ item.